### DISCUSSION TITLE
Sending Control + f5 keys during the test execution

### DISCUSSION THREAD

**Discussion by jasminhusadzic** [webdriverio/webdriverio#7052] [Q&A]
I would like to hear opinion how to perform this action. I have specific flow where i need to pass Control + f5 combination. Was try with browser.keys, but without success. Any idea?

**Comment by christian-bromann** [comment]
`browser.keys(['Control', 'F5'])` would be my guess.

**Reply by jasminhusadzic** [reply]
Thanks for quick reply. Not working.
Also try:
`browser.keys(['Control', 'F5', 'NULL'])` and ` browser.keys(['\ue009', '\ue035'])`

Actual behavior: Nothing happen

**Reply by christian-bromann** [reply]
What environment/browser? Devtools or WebDriver?

**Reply by jasminhusadzic** [reply]
Chrome, WebDriver and Selenium Standalone.
